tests/php/sync/test_class.jetpack-sync-lock.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tests for the Lock class.
+ *
+ * @package automattic/jetpack-sync
+ */
+
+use Automattic\Jetpack\Sync\Lock;
+
+/**
+ * WP_Test_Jetpack_Sync_Lock class
+ */
+class WP_Test_Jetpack_Sync_Lock extends WP_Test_Jetpack_Sync_Base {
+
+	/**
+	 * Test request lock twice
+	 */
+	public function test_request_lock_twice() {
```

Comment:
Do we want to test that we can have multiple locks?

Reply:
with different names you mean?

Reply:
https://github.com/Automattic/jetpack/pull/14039/commits/0410271d8477da5fb93a85a2f3d38a669b09583d